components/PaginationLinks.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Link from "next/link";
+
+const PaginationLinks = ({offset, username}) => {
+  const offset2 = offset || 0;
+  let olderLink = '/' + username + '?offset=' + (+offset2 + 30);
+  let newerLink = '/' + username + '?offset=' + (+offset2 - 30);
+
+  return <>
+    <ul className="pagination">
+      <li className="newer">
+        { offset2 > 0 &&
+        <Link href={newerLink}>
+          ← Newer entries
+        </Link>
+        }
```

Comment:
To align with the style elsewhere, let's use it with parens, like this:

```
{blah blah && (
  <Blah/>
)}
```

Or we can just compact it in a single line, without parens.